src/ci/docker/scripts/x86_64-gnu-llvm.sh

```diff
@@ -12,6 +12,10 @@ if [[ -z "${PR_CI_JOB}" ]]; then
         ../x.py --stage 1 test --skip src/tools/tidy
     fi
 
+    # The tests are run a second time with the size optimizations enabled.
```

Comment:
Sorry, realized one more thing. Since we change rustc flags here, possibly everything will be recompiled. Could you please move this test invocation after line `29`, so that the following two test invocations don't unnecessarily suffer from an additional rebuild? Thanks.